css/about.css

```diff
@@ -0,0 +1,206 @@
+/* http://meyerweb.com/eric/tools/css/reset/
```

Comment:
Nice work splitting the about.css and index.css into different files.  I like doing it this way.  Otherwise I have to search a full .css file to find the the styles for multiple pages.